src/WebJobs.Extensions.OpenAI/Search/SemanticSearchAttribute.cs

```diff
@@ -40,14 +40,24 @@ public SemanticSearchAttribute(string connectionName, string collection)
     public string ConnectionName { get; set; }
 
     /// <summary>
-    /// The name of the collection or table to search.
+    /// The name of the collection or table or index to search.
     /// </summary>
     /// <remarks>
     /// This property supports binding expressions.
     /// </remarks>
     [AutoResolve]
     public string Collection { get; set; }
 
+
+    /// <summary>
+    /// The api key for client authentication, pass null for managed identity.
```

Comment:
This description shouldn't make assumptions about the implementation. For example, future search providers may not support API keys and may not support managed identity. I suggest a more generic description, like:

```suggestion
    /// The name of the app setting or environment variable containing the required credentials for authenticating with the search provider. See the documentation for the search provider extension to know what format the underlying credential value requires.
```